pallets/rc-migrator/src/lib.rs

```diff
@@ -365,18 +377,75 @@ pub mod pallet {
 	#[pallet::pallet]
 	pub struct Pallet<T>(_);
 
+	#[pallet::call]
+	impl<T: Config> Pallet<T> {
+		/// Set the migration stage.
+		///
+		/// This call is intended for emergency use only and is guarded by the
+		/// [`Config::ManagerOrigin`].
+		#[pallet::call_index(0)]
+		pub fn set_stage(origin: OriginFor<T>, stage: MigrationStageOf<T>) -> DispatchResult {
+			<T as Config>::ManagerOrigin::ensure_origin(origin)?;
+			Self::transition(stage);
+			Ok(())
+		}
+
+		/// Schedule the migration to start at a given moment.
+		#[pallet::call_index(1)]
+		pub fn schedule_migration(
+			origin: OriginFor<T>,
+			start_moment: DispatchTime<BlockNumberFor<T>>,
+		) -> DispatchResult {
+			<T as Config>::ManagerOrigin::ensure_origin(origin)?;
+			let now = frame_system::Pallet::<T>::block_number();
+			let block_number = start_moment.evaluate(now);
+			Self::transition(MigrationStage::Scheduled { block_number });
+			Ok(())
+		}
+
+		/// Start the data migration.
+		///
+		/// This is typically called by the Asset Hub to indicate it's readiness to receive the
+		/// migration data.
+		#[pallet::call_index(2)]
+		pub fn start_data_migration(origin: OriginFor<T>) -> DispatchResult {
```

Comment:
```suggestion
		pub fn ah_ready_for_data(origin: OriginFor<T>) -> DispatchResult {
```
? I think it could help to include `ah` in there to show that it comes from the AH.

Reply:
I think can be more general, we might include more. But also no strong opinion. I just trying to not create too many stages to not waste time.

Reply:
Okay we can rename when it needs to be more specific later.